src/PowerShellEditorServices/Session/SessionDetails.cs

```diff
@@ -56,7 +56,7 @@ public static PSCommand GetDetailsCommand()
         {
             PSCommand infoCommand = new PSCommand();
             infoCommand.AddScript(
-                "@{ 'computerName' = $env:ComputerName; 'processId' = $PID; 'instanceId' = $host.InstanceId }");
+                "@{ 'computerName' = if ($ExecutionContext.Host.Runspace.ConnectionInfo) {$ExecutionContext.Host.Runspace.ConnectionInfo.ComputerName}  else {'localhost'}; 'processId' = $PID; 'instanceId' = $host.InstanceId }");
```

Comment:
When connected to a remote session on Linux or macOS, does ComputerName contain anything?  Aside from that I think this change is fine

Reply:
So, when I tested on linux `ConnectionInfo` was null. 

I could include a check for `ComputerName` being null to be safe.

Reply:
In a remote session to a Linux machine ConnectionInfo was null?  Pretty unfortunate if so, though I suppose `$env:ComputerName` never worked on Linux anyway.

Reply:
I'm pretty sure the only time `ConnectionInfo` is set is when it's `WSManConnectionInfo`.  Frustratingly, even `NamedPipeConnectionInfo` never gets set 😕

Reply:
What about `[Environment]::MachineName`? Looks like it works as expected on my Ubuntu box at least.  Maybe something like this just in case it's empty on some distros

```powershell
if ([string]::IsNullOrWhitespace([Environment]::MachineName)) {
    'localhost'
} else {
    [Environment]::MachineName
}

Reply:
Yup, that works on my Ubuntu instance as well - 16.04.

Reply:
Yep, `[Environment]::MachineName` seems good to me too, but I'd check to make sure it also works on PowerShell 3-5.1 just in case.  I dunno if we can assume 'localhost' as default since we also run this on remote sessions.  I think if it turns up empty we're in trouble no matter what :)

Reply:
`[Environment]::MachineName` works on macOS and 5.1 too!

@daviwil I thought PSRP in vscode-powershell wasn't allowed in PS 3 and 4?

Reply:
Remoting works in PS4, but we should make sure that it works on the local machine for PS3/4 too.  Though if it works in PS 5.1, I would assume it works in PS 3/4 too.

Reply:
PS 3/4 have `[Environment]::MachineName` 🎉

Let's go with that.

Reply:
Ok I changed it to `[Environment]::MachineName`. I kept the `if` just in case. Let me get some 👍's from you guys and I'll merge this it! 😃